### ISSUE TITLE
ASP.net issues with button!

### ISSUE THREAD

**Issue by oliver-chalk** [fancyapps/fancybox#345]
Hi all, 

I have a bit of a weird one with asp.net (vb) and trying to open an iframe using an asp:button!

This is the closest I've come, all the latest files and all proper JS added/loaded!
This is my js that I've linked to in the Site.Master: 

$(document).ready(function() {
    $('.savingorloading-iframe').fancybox({
                type        : 'iframe',
                maxWidth    : 230,
        maxHeight   : 50,
        fitToView   : false,
        width       : '80%',
        height      : '80%',
        autoSize    : false,
        closeBtn    : false,
        helpers:  {
        title : {
            type : 'inside'
        },
        overlay : {
            opacity : 0.2
        }
    }
    });
});

I have this at the top of my nested page:

script type="text/javascript">
        function ClickLoadButton() {
            //$('.savingorloading-iframe').fancybox.open();
            $('[id$=HiddenLoadLink]').click(); 
        }
</script

In that same nested page, I have three links in this order:

a id="HiddenLoadLink" class="savingorloading-iframe" data-fancybox-type="iframe" href="Scripts/loading.html">Link1</a 
This one works as expected, brings up the 'loading.html' page.

a href="#" onclick="return ClickLoadButton();" >Link2</a
Again, this one works as expected, brings up the 'loading.html' page, so the javascript function to the above hyperlink is working.

However this:
asp:Button ID="SubmitFilter" runat="server" Text="Update" OnClientClick="return ClickLoadButton();" /
Brings up the same as above but with no content before doing the buttons behind code? 
Using firebug and google dev tools it is clicking the hidden link?!

Basically I have a submit button and on click I want it to load "scripts/loading.html" in a iframe popup so the users know the page is loading/saving, before it submits and does what the button is suppose to do in the behind code.
Any idea's I'd be very grateful as this is now bugging me!

Thanks,
Ollie
